pkg/cmd/cli/install/install.go

```diff
@@ -72,6 +76,10 @@ func (o *InstallOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.VeleroPodMemRequest, "velero-pod-mem-request", o.VeleroPodMemRequest, "memory request for Velero pod. Optional.")
 	flags.StringVar(&o.VeleroPodCPULimit, "velero-pod-cpu-limit", o.VeleroPodCPULimit, "CPU limit for Velero pod. Optional.")
 	flags.StringVar(&o.VeleroPodMemLimit, "velero-pod-mem-limit", o.VeleroPodMemLimit, "memory limit for Velero pod. Optional.")
+	flags.StringVar(&o.ResticPodCPURequest, "restic-pod-cpu-request", o.ResticPodCPURequest, "CPU request for Restic pod. Optional.")
```

Comment:
"restic pods" since it's for the daemonset (same for all 4 flags).

Reply:
@skriss are you just referring to the description here, or should the flag also be plural? I think it might be easier to keep the flag singular as that will be consistent with the velero ones?

Reply:
Ah - yeah, I think just updating the description is fine.